examples/getCiphers/getCiphers.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
since this example calls into the ipmitool provider directly, I suggest naming the parent directory, `ipmitool-getCiphers`